docs/how-to/partitions.rst

```diff
@@ -0,0 +1,136 @@
+******************************************
+Add partition support to an application
+******************************************
+
+Partitions basics
+=================
+
+In the below examples, we work with three partitions: ``default``, ``kernel``, and ``bar-baz``.  The ``default`` and ``kernel`` partitions do not have a namespace.  The ``bar-baz`` partition is part of the ``component`` namespace.
```

Comment:
Maybe calling the partitions `default`, `foo` and `bar` and namespacing `bar` to have `bar-baz` inside could make example names more consistent and easier to follow?